### PR TITLE
Added support for web components in screenshot test

### PR DESCRIPTION
**Pull request by dwnoble**
Updated `charts_rendered` helper to wait for the web components as well as charts. The original check wasn't working for web components because selenium's element selectors can't reach into the web component shadow root

### CODE REVIEW

File: server/webdriver/shared.py [hunk 1]
```diff
@@ -23,6 +23,11 @@
 ASYNC_ELEMENT_HOLDER_CLASS = 'dc-async-element-holder'
 ASYNC_ELEMENT_CLASS = 'dc-async-element'
 TIMEOUT = 60
+WEB_COMPONENT_TAG_NAMES = [
```
Comment (shifucun) [inline server/webdriver/shared.py:26]: Add a comment in here and the components definition page to sync both places?

**Review by shifucun — APPROVED** [review]
Thanks! Can you run ./run_test.sh -s locally to verify?

File: server/webdriver/shared.py (hunk 1 above)
Comment (dwnoble) [reply server/webdriver/shared.py:28]: done!

**Comment by dwnoble** [comment]
> Thanks! Can you run ./run_test.sh -s locally to verify?

Verified the tests pass locally

**Pull request merged**